source

```diff
@@ -102551,7 +102551,8 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
    <dd>Switch to the <span>attribute value (single-quoted) state</span>.</dd>
 
    <dt>U+003E GREATER-THAN SIGN (&gt;)</dt>
-   <dd><span>Parse error</span>. Treat it as per the "anything else" entry below.</dd>
+   <dd>This is an <dfn data-x="parse-error-unexpected-character-in-attribute-value">unexpected-character-in-attribute-value</dfn>
```

Comment:
After reconsumption it will actually close the tag. So the issue here is that attribute is missing value:
```html
<div id=>
```
I suggest to raise `missing-attribute-value` error and add `Switch to the data state. Emit the current tag token.` instead of reconsuption.